src/blocked.c

```diff
@@ -430,6 +430,10 @@ void serveClientsBlockedOnStreamKey(robj *o, readyList *rl) {
                                              receiver->bpop.xread_consumer->ptr,
                                              SLC_NONE,
                                              &created);
+                    if (created) {
+                        notifyKeyspaceEvent(NOTIFY_STREAM,"xgroup-createconsumer",
+                                            rl->key,rl->db->id);
+                    }
```

Comment:
maybe push it into `streamLookupConsumer`?

Reply:
(and pass the keyname and dbid as additional args)

Reply:
done. plz review again.